### PR TITLE
Fix addbot Tcl command

### PR DESCRIPTION
**Pull request by vanosg**
Found by: Geo
Patch by: Geo
Fixes: #470

One-line summary:
Fix addbot Tcl command

Additional description (if needed):

Test cases demonstrating functionality (if applicable):
```
.tcl addbot foo 1.1.1.1/3333/4444
Tcl: 1
.whois foo
[07:03:40] #-HQ# whois foo
HANDLE                           PASS NOTES FLAGS           LAST
foo                              no       0 b               never (nowhere)
  ADDRESS: 1.1.1.1
     users: 4444, bots: 3333
```
```
.tcl addbot foo 1.1.1.1:4444/5555
Tcl: 1
.whois foo
[07:04:21] #-HQ# whois foo
HANDLE                           PASS NOTES FLAGS           LAST
foo                              no       0 b               never (nowhere)
  ADDRESS: 1.1.1.1
     users: 5555, bots: 4444
```
```
.tcl addbot foo \[fe80::20c:29ff:fea7:5864\]:4444
Tcl: 1
.whois foo
[07:04:56] #-HQ# whois foo
HANDLE                           PASS NOTES FLAGS           LAST
foo                              no       0 b               never (nowhere)
  ADDRESS: [fe80::20c:29ff:fea7:5864]
     users: 4444, bots: 4444
```
See Jenkins tests for full combinatorial.

### CODE REVIEW

**Comment by vanosg** [comment]
Build finished.

File: doc/sphinx_source/mainDocs/tcl-commands.rst [hunk 1]
```diff
@@ -322,7 +322,13 @@ adduser <handle> [hostmask]
 ^^^^^^^^^^^^^^^^^^^^^^^^^
 addbot <handle> <address>
 ^^^^^^^^^^^^^^^^^^^^^^^^^
-  Description: adds a new bot to the userlist with the handle and botaddress given (with no password and no flags)
+  Description: adds a new bot to the userlist with the handle and botaddress given (with no password and no flags). <address> format is one of:
+
+  - ipaddress:botport/userport
```
Comment (Cizzle) [inline doc/sphinx_source/mainDocs/tcl-commands.rst]: Shouldn't this be "ipv4address"?

File: doc/sphinx_source/mainDocs/tcl-commands.rst [hunk 2]
```diff
@@ -322,7 +322,13 @@ adduser <handle> [hostmask]
...
+  - ipaddress/botport/userport
+  - [ipv6address]:botport/userport
+
+  NOTE: The ipaddress:botport/userport format is depreciated and will be removed in a future version. It is recommended to use the other two formats.
```
Comment (Cizzle) [inline doc/sphinx_source/mainDocs/tcl-commands.rst]: I think "deprecated" is more correct? And isn't the "[ipv6address]:" going to be deprecated too?

Comment (vanosg) [reply doc/sphinx_source/mainDocs/tcl-commands.rst]: I'm excited that the only two comments on one of my PRs were grammatical, not code-based :)

**Comment by vanosg** [comment]
Build finished.

**Comment by Cizzle** [comment]
LGTM

**Comment by vanosg** [comment]
Build finished.

**Pull request merged**